lib/manager/components/ProjectViewer.js

```diff
@@ -175,72 +176,81 @@ export default class ProjectViewer extends Component<Props, State> {
+                  {project.autoFetchFeeds
+                    ? `${project.autoFetchHour}:${
+                      project.autoFetchMinute < 10
+                        ? '0' + project.autoFetchMinute
+                        : project.autoFetchMinute
+                    }`
+                    : 'Auto fetch disabled'}
+                </li>
               </ul>
             </Col>
           </Row>
           <Tabs
             id='project-viewer-tabs'
             activeKey={activeComponent || 'sources'}
             mountOnEnter
-            onSelect={this._selectTab}>
+            onSelect={this._selectTab}
+          >
             <Tab
               eventKey='sources'
               title={
                 <span>
                   <Glyphicon className='icon-link' glyph='list' />
-                  <span className='hidden-xs'>{this.messages('feeds.title')}</span>
+                  <span className='hidden-xs'>
+                    {this.messages('feeds.title')}
+                  </span>
                 </span>
-              }>
+              }
+            >
               <Row>
                 <Col xs={12} sm={10}>
-                  {this.state.createMode
-                    ? (
-                      <CreateFeedSource
-                        createFeedSource={createFeedSource}
-                        onCancel={this._toggleCreateView}
-                        projectId={project.id}
-                      />
-                    )
-                    : (
-                      <FeedSourceTable
-                        onNewFeedSourceClick={this._toggleCreateView}
-                        project={project}
-                      />
-                    )
-                  }
+                  {this.state.createMode ? (
+                    <CreateFeedSource
+                      createFeedSource={createFeedSource}
+                      onCancel={this._toggleCreateView}
+                      projectId={project.id}
+                    />
+                  ) : (
+                    <FeedSourceTable
+                      onNewFeedSourceClick={this._toggleCreateView}
+                      project={project}
+                    />
+                  )}
                 </Col>
                 <Col xs={12} sm={2}>
                   {this._renderPublicFeeds()}
                   <ProjectSummaryPanel
                     project={project}
-                    feedSources={project.feedSources || []} />
-                  <Panel header={<h3>What is a feed source?</h3>}>
-                    A feed source defines the location or upstream source of a{' '}
-                    GTFS feed. GTFS can be populated via automatic fetch,{' '}
-                    directly editing or uploading a zip file.
-                  </Panel>
+                    feedSources={project.feedSources || []}
+                  />
+                  <ExplanatoryPanel project={project} />
+                  <LabelPanel project={project} user={user} />
```

Comment:
Labels in the ProjectViewer should not be this tall, even with the label Edit and Delete buttons visible.

Reply:
New CSS shrinks them a little, but am open to more feedback on the matter.

Reply:
Make the text on these the same size as in the labels filter dropdown, or the label Edit button. At the current size, the labels feel a bit overwhelming compared to the other content.